cdp_scrapers/instances/portland.py

```diff
@@ -257,23 +359,27 @@ def get_event_minutes(
 
         # think we will let MinutesItem.name = Matter.name
         #                   MinutesItem.description = Matter.title
+
+        minute_sections = event_page.find_all(
+            "div", class_="relation--type-agenda-item"
+        )
+        event_minute_items = []
+        for minute_section in minute_sections:
```

Comment:
As mentioned in [this comment](https://github.com/CouncilDataProject/cdp-scrapers/pull/70/files#r790387749) I think this `for` loop can be replaced with list comprehension. In fact the loop and the `return` line can be combined to something like

```Python
return reduced_list(
    [
        self.get_none_if_empty(
            EventMinutesItem(
                index=int(minute_section.find("h4").text.strip()),
                ...,
            )
            for minute_section in minute_sections
        )
    ]
)
```

I could be wrong but I think it works...

Reply:
I don't exactly know how I can implement this since I need the following three lines of code for each `minute_section`:
`title_div=minute_section.find("div", class_="council-document__title")`
`matter_title = title_div.find("a").text.strip()`
`index = int(minute_section.find("h4").text.strip())`

Reply:
I see what you mean. It would be something like this.

```Python
return reduced_list(
    [
        self.get_none_if_empty(
            EventMinutesItem(
                index=int(minute_section.find("h4").text.strip()),
                matter=self.get_matter(minute_section),
                minutes_item=self.get_none_if_empty(
                    MinutesItem(
                        description=minute_section.find("div", class_="council-document__title").find("a").text.strip(),
                    )
                ),
            )
        )
    ]
)
```

Now, I agree with you for this blob, it may be more readable to leave it as you have it, in conventional `for` loop form. So let's leave this for now. We can revisit later if we wish.